### PR TITLE
Service to get container information from batch code.

### PR DESCRIPTION
**Pull request by samanthamunoz**
service that takes in a batch code and gives all relevant container info

### CODE REVIEW

File: modules/ServerAPI/src/server/routes/InventoryServiceRoutes.coffee [hunk 1]
```diff
@@ -2304,3 +2305,43 @@ formatContainers = (containers, modifiedBy, modifiedDate, callback) ->
...
+	queryPayload =
+		{
+			"containerType": "well",
+			"containerKind": "default",
+			"stateType": "status",
+			"stateKind": "content",
+			"valueType": "codeValue",
+			"valueKind": "batch code",
+			"value": requestObject.batchCode,
+			"maxResults": 600,
```
Comment (brianbolt) [inline modules/ServerAPI/src/server/routes/InventoryServiceRoutes.coffee]: I thought we were going to make max results a config

Comment (samanthamunoz) [inline modules/ServerAPI/src/server/routes/InventoryServiceRoutes.coffee]: @brianbolt - where are configs pulled in from in base acas, same place as custom?

**Comment by brianbolt** [comment]
This file is where all the default configs are set:
https://github.com/mcneilco/acas/blob/master/conf/config.properties.example

Client's can override these configs with their own config files.

**Comment by samanthamunoz** [comment]
@brianbolt - this has been updated, ready for your review.

**Pull request merged**